src/py/flwr/server/superlink/state/in_memory_state.py

```diff
@@ -40,6 +40,9 @@ def __init__(self) -> None:
         self.run_ids: Dict[int, Tuple[str, str]] = {}
         self.task_ins_store: Dict[UUID, TaskIns] = {}
         self.task_res_store: Dict[UUID, TaskRes] = {}
+        self.client_public_keys: Set[bytes] = set()
+        self.server_public_key: bytes = b""
+        self.server_private_key: bytes = b""
```

Comment:
Would it make sense to use `Optional[bytes]` here instead of an empty `str`?

Reply:
Yes, I made it optional.